pkg/app/piped/executor/scriptrun/scriptrun.go

```diff
@@ -0,0 +1,68 @@
+package scriptrun
```

Comment:
Please add the file header.

```
// Copyright 2023 The PipeCD Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

```

Reply:
Applied! ✅ 
[5525832](https://github.com/pipe-cd/pipecd/pull/4720/commits/5525832a0bb5572b29a1e9cb19510ec4b89d48b5)